daisy/compute/compute.go

```diff
@@ -350,7 +364,7 @@ func (c *client) operationsWaitHelper(project, name string, getOperation operati
 
 		switch op.Status {
 		case "PENDING", "RUNNING":
-			time.Sleep(1 * time.Second)
+			time.Sleep(5 * time.Second)
```

Comment:
do we still need this? If I remember correctly, this was done to reduce the frequence of wait operations? With v.0.44.0 GCP go library, we might not need to do this.

Reply:
No we don't. Updated the PR to revert it back to 1 second.